examples/timestamping/backend/src/schema.rs

```diff
@@ -42,13 +42,14 @@ encoding_struct! {
     }
 }
 
+/// Timestamping information schema.
 #[derive(Debug)]
 pub struct Schema<T> {
     view: T,
 }
 
-/// Timestamping information schema.
 impl<T> Schema<T> {
+    /// Creates new timestamping schema with given snapshot.
```

Comment:
Well, actually, not only snapshot, `Fork` can also be passed here.
`Creates a new schema with a given snapshot`